tools/pythonpkg/tests/fast/pandas/test_timedelta.py

```diff
@@ -47,6 +47,7 @@ def test_timedelta_negative(self, duckdb_cursor):
     @pytest.mark.parametrize('minutes', [0, 60])
     @pytest.mark.parametrize('hours', [0, 24])
     @pytest.mark.parametrize('weeks', [0, 51])
+    @pytest.mark.skipif(platform.system() == "Emscripten", reason="Bind parameters are broken when running on Pyodide")
```

Comment:
The error message is a bit coarse-grained here:

What fails is:

```python
con.execute("SELECT $2::date - $1::date", ['2024-01-02', '2024-03-04'])
```

Reply:
Can you expand on this?

It seems to be working on https://duckdb.github.io/duckdb-pyodide/console by doing:
```
import duckdb
duckdb.execute("SELECT $2::date - $1::date", ['2024-01-02', '2024-03-04'])
```
unsure if the problem is in the testing (possibly due to handling of timezones / else) or this is actually broken.

Reply:
Sorry, try the code from the test this is marking.

Regardless, I was hoping this could be fixed in a follow-up, as this bug existed before we were running the test suite, so adding the tests is not making anything worse 😅

Reply:
@carlopi Here's an example that fails:

```python
import duckdb
days = 1
weeks = 2
seconds = 3
microseconds = 4
milliseconds = 5
minutes = 6
hours = 7
instant = f"""
                (INTERVAL {days + (weeks * 7)} DAYS +
                INTERVAL {seconds} SECONDS +
                INTERVAL {microseconds} MICROSECONDS +
                INTERVAL {milliseconds} MILLISECONDS +
                INTERVAL {minutes} MINUTE +
                INTERVAL {hours} HOURS)
            """
query = f"select '1990/02/11'::DATE - {instant}, '1990/02/11'::DATE - $1"
con = duckdb.connect()

import datetime
val = datetime.timedelta(days,seconds,microseconds,milliseconds,minutes,hours,wee
ks)

con.execute(query, [val])
```

Reply:
The error I get is:

```
Pyodide has suffered a fatal error. Please report this to the Pyodide maintainers.
The cause of the fatal error was:
TypeError: Cannot convert undefined to a BigInt
Look in the browser console for more details.
```